<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeItemInfo</class>
 <widget class="QDialog" name="ChangeItemInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>20</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Change Item Information</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Title:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="find">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Author:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="isbnLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="subjectLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cisbn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change ISBN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ctitle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="csubject">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Subject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cauthor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Author</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
